src/js/pages/OpenDispute/index.jsx

```diff
@@ -0,0 +1,120 @@
+import React, {Component} from 'react';
+import Textarea from 'react-validation/build/textarea';
+import Form from 'react-validation/build/form';
+import {Button} from 'reactstrap';
+import Loading from '../../components/Loading';
+import ConfirmDialog from '../../components/ConfirmDialog';
+import PropTypes from 'prop-types';
+import {withRouter} from "react-router-dom";
+import {connect} from "react-redux";
+import arbitration from '../../features/arbitration';
+import successImage from '../../../images/success.png';
+
+class OpenDispute extends Component {
+  state = {
+    value: '',
+    displayDialog: false
+  }
+
+  constructor(props){
+    super(props);
+    props.loadArbitration(props.escrowId);
+  }
+
+  componentDidUpdate() {
+    if (this.props.escrow.arbitration) {
+      if(this.props.escrow.arbitration.open || this.props.escrow.arbitration.result !== "0"){
+        this.props.history.push('/');
+      }
+    }
+  }
+
+  handleChange = (e) => {
+    this.setState({value: e.target.value});
+  }
+
+  displayDialog = show => () => {
+    this.setState({displayDialog: show});
+  };
+
+  goToProfile = () => {
+    this.props.history.push('/profile');
+  }
+
+
+  render(){
+    const {escrow, openDispute, loading, receipt} = this.props;
+
+    if(!escrow) return <Loading page={true} />;
+    if(loading) return <Loading mining={true} />;
+
+    if(receipt) return (
+      <div className="text-center p-5">
+        <img src={successImage} alt="Success" width="160" height="160" className="mt-5" />
+        <h2 className="mt-5">Your dispute was successfully open</h2>
+        <p className="text-muted">Follow the progress of your dispute in the profile.</p>
+        <p>
+          <Button color="primary" onClick={this.goToProfile}>Okay</Button>
+        </p>
+    </div>
+    );
+
+    const onClickDialog = () => {
+      openDispute(escrow.escrowId);
+    };
```

Comment:
This function can be put outside of `render` so that is not defined every render